tools/detekt/src/main/kotlin/com/datadog/tools/detekt/rules/InvalidStringFormat.kt

```diff
@@ -291,12 +292,13 @@ class InvalidStringFormat : Rule() {
             "%(\\d+\\$)?([flags]+)?(\\d+)?(\\.\\d+)?([sbhcdoxegfa])"
         )
 
+        private const val INDEX_REF = 1
+
         // TODO check flags, width and precision for invalid use?
-        private val INDEX_REF = 1
-        private val INDEX_FLAGS = 2
-        private val INDEX_WIDTH = 3
-        private val INDEX_PRECISION = 4
-        private val INDEX_TYPE = 5
+//        private const val INDEX_FLAGS = 2
+//        private const val INDEX_WIDTH = 3
+//        private const val INDEX_PRECISION = 4
```

Comment:
should commented stuff be deleted?

Reply:
I'm leaving it here on purpose as it goes with the TODO and helps understand why the REF and TYPE indices are 1 and 5.